src/index.html

```diff
@@ -17,6 +17,93 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#home"
+        class="logo_link"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Moyo logo"
+          class="logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
```

Comment:
The class `nav_item` should be `nav__item` to follow the BEM naming convention.

Reply:
👍